integration-tests/tests/src/tests/iterators.ts

```diff
@@ -159,28 +158,43 @@ describe("Iterating", () => {
   }
 
   describe("unfiltered results", () => {
-    itCanIterate(() => realm.objects<IPerson>("Person"), ["Alice", "Bob", "Charlie"]);
+    itCanIterate(
+      function (this: RealmContext) {
+        return this.realm.objects<IPerson>("Person");
+      },
+      ["Alice", "Bob", "Charlie"],
+    );
   });
 
   describe("filtered results", () => {
-    itCanIterate(() => realm.objects<IPerson>("Person").filtered("age > 60"), ["Charlie"]);
+    itCanIterate(
+      function (this: RealmContext) {
```

Comment:
Curious what the reasoning is behind using a `function` declaration instead of an arrow function?

Reply:
Ah, I think I remember us having a discussion on this a while back.  That's a shame, but seems like a necessary evil :)

Reply:
Yeah, I don't love it (or the Mocha context stuff in general) but it serves a purpose, heh

Reply:
Maybe adding https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-mocha-arrows.md in future would make sense